botocore/data/s3/2006-03-01/service-2.sdk-extras.json

```diff
@@ -0,0 +1,11 @@
+{
+  "version": 1.0,
+  "merge": {
+    "metadata":{
+      "protocol":"s3"
```

Comment:
There currently also exists a `"protocols":["rest-xml"],` value in the s3 service model. We should probably update this too. I don't think it's used anywhere yet though.